tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -268,3 +268,29 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version'))) {
     )
   });
 }
+
+
+if (!/\/2\.7/.test(Cypress.env('rancher_version'))) {
+  describe('Test OCI support', { tags: '@p1'}, () => {
+    qase(60,
+      it("Fleet-60: Test OCI helm chart support on Github Container Registry", { tags: '@fleet-60' }, () => {;
+        const repoName = 'default-oci-60'
+        const repoUrl = 'https://github.com/rancher/fleet-test-data'
+        const branch = 'master'
+        const path = 'helm-oci'
+
+        cy.fleetNamespaceToggle('fleet-default');
+        cy.addFleetGitRepo({ repoName, repoUrl, branch, path });
+        cy.clickButton('Create');
+        cy.verifyTableRow(0, 'Active', '1/1');
+        cy.accesMenuSelection('k3d-imported', 'Storage', 'ConfigMaps');
+        cy.nameSpaceMenuToggle('All Namespaces');
+        cy.filterInSearchBox('fleet-test-configmap');
+        cy.get('.col-link-detail').contains('fleet-test-configmap').should('be.visible').click({ force: true });
+        cy.get('section#data').should('contain', 'default-name').and('contain', 'value');
```

Comment:
can this be in a single function? I mean more generalized for e.g. Pod, Deployment, Secret, ConfigMaps matching etc.
Please let me know if it is not clear. Happy to pair with you to discuss it. :smile:

Reply:
Thanks for the recommendation. I had an initial thought of trying but I had the feeling I'd be overencapsulating. Still, there is a problem with locator `section#data` as it is not universal accross options.
If ok, I'd rather push this now and I will create a ticket for attempting refactoring this better.